drone/repo_chown.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/codegangsta/cli"
+)
+
+var repoChownCmd = cli.Command{
+	Name:  "chown",
+	Usage: "assume ownership of a repository",
+	Action: func(c *cli.Context) {
+		if err := repoChown(c); err != nil {
+			log.Fatalln(err)
+		}
+	},
+}
+
+func repoChown(c *cli.Context) error {
+	repo := c.Args().First()
+	owner, name, err := parseRepo(repo)
+	if err != nil {
+		return err
+	}
+
+	client, err := newClient(c)
+	if err != nil {
+		return err
+	}
+
+	if _, err := client.RepoChown(owner, name); err != nil {
+		return err
+	}
+	fmt.Printf("Successfully assumed ownership of repository %s/%s\n", owner, name)
```

Comment:
I've gotten feedback that people want some sort of printed message, so I think this is good. We can eventually add `--silent` for people that want to use the CLI in scripting